src/transformStateWithClones.js

```diff
@@ -7,7 +7,43 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const states = [];
+  let newState = {
+    ...state,
+  };
+
+  for (const i of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
Давай переменным понятные имена, `i` подходит для счетчика(от слова `iterator`), но не подходит для объекта